app/models/edition.rb

```diff
@@ -702,6 +702,25 @@ def has_legacy_tags?
     has_primary_sector? || has_secondary_sectors?
   end
 
+  # For more info, see https://docs.publishing.service.gov.uk/manual/content-preview.html#authentication
```

Comment:
We should add tests for both the methods added in here to the edition spec. For the auth_bypass_token, you might be able to take some inspiration from [these tests](https://github.com/alphagov/content-publisher/blob/8df63907dab486a54894105b829ab9ea02936b67/spec/models/edition_spec.rb#L203) in content publisher